be2-scala/src/main/scala/ch/epfl/pop/model/objects/Channel.scala

```diff
@@ -11,10 +11,10 @@ final case class Channel(channel: String) {
    *
    * @return an array of bytes corresponding to the decoded sub-channel name or None if an error occurred
```

Comment:
the comments here and below might need updating

Reply:
thank you, I had missed that